src/Storage/Configuration/RegisterServiceSettings.cs

```diff
@@ -1,26 +1,14 @@
-using System.Runtime.Serialization;
-
 namespace Altinn.Platform.Storage.Configuration
 {
     /// <summary>
     /// Represents a set of configuration options when communicating with the platform API.
     /// Instances of this class is initialised with values from app settings. Some values can be overridden by environment variables.
     /// </summary>
-    public class PlatformSettings
+    public class RegisterServiceSettings
```

Comment:
We must also change the name of the environment variables in the ops project.

Reply:
You are correct! 💡 👍 Was waiting for the approval of this PR. Now will have a look at the environment variable in the ops project! 🙂